index.html

```diff
@@ -28,6 +26,138 @@ <h3 class="job-title">I'm a Software Developer</h3>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project4 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project5 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project6 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project7 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    </section>
+                    <button type="button" class="full-btn">See Project</button>
```

Comment:
- [ ] Kindly make sure you removed this `button`, because it is not the part of the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=BdCnAzByz0Gs7hlS-0)

![btn](https://user-images.githubusercontent.com/57408419/227956256-a0501417-394c-449c-90d8-0c9b0c878205.JPG)